project/src/components/cities-map/cities-map.jsx

```diff
@@ -41,17 +67,24 @@ function CitiesMap({ city, offers, className = '' }) {
 
       map.current.setView([ city.location.latitude, city.location.longitude ], city.location.zoom);
 
-      offers.forEach((o) => {
+      offers.forEach((o, idx) => {
         const coords = [
           o.location.latitude,
           o.location.longitude,
         ];
-        leaflet
-          .marker(coords, {icon})
-          .addTo(map.current);
+        if (activeOfferId === o.id) {
+          leaflet
+            .marker(coords, {activeIcon})
```

Comment:
Это было причиной ошибки:
```suggestion
            .marker(coords, {icon: activeIcon})
```

Reply:
Ага, невнимательно было)